accepted/2025/cli-acquisition-tool.md

```diff
@@ -221,3 +236,111 @@ _Where_ the SDK is resolved from is largely orthogonal to _how compatible_ that
 The SDK team plans to release an initial version of `dnup` covering environment setup and basic install functionality as soon as possible. We then want to gather feedback from users and iterate on the feature set until the tool is a consistent, one-stop-shop for acquiring and managing .NET tooling of all kinds.
 
 The end goal is to stabilize the core workflows and experiences enough to justify the tool being as assumed part of the `dotnet` CLI's tooling capabilities - we want the SDK to be able to rely on `dnup` and use it to orchestrate more complex workspace-wide gestures like a toolset-level `dotnet restore` kind of operation that gets all of the SDKs and Runtimes needed for a workspace.
+
+## Milestones
+
+This is a large effort, and there are different areas of the work that will progress at different speeds. In addition, for compliance reasons we need to ensure that the tool is secure and robust before we can recommend it for broad usage. As a result, we expect to have multiple milestones along the way. Each of these milestones will likely be usable by motivated users, be we will not recommend them for broad usage until we reach the Public Preview milestone.
+
+* **MVP**
```

Comment:
I personally would prefer to name this something different.  To me a true MVP is what we should release to the public.